gwu-libraries/scholarspace-hyrax
# T416 missing keywords

Completes fix of #416 

Adds a Rake task that identifies GwETD works with a null string in the keyword field (`[""]`) and resets that field to the proper null (`[]`). 

To test:

1. Create some ETD's with and without keywords.
2. In the Rails console, using `GwEtd.find`, retrieve one or more of the works lacking keywords, and add the null string:
```
gw_etd.keyword = [""]
gw_etd.save
```
3. From the Hyrax app container, run `rake gwss:delete_null_keywords`.
4. Confirm that the work(s) where you set the `keyword` field to the null string do not display that field in the UI, and that the `Keywords` field is present for the others.
```
